aws-lc-rs/src/signature.rs

```diff
@@ -370,6 +376,11 @@ impl<B: AsRef<[u8]>> UnparsedPublicKey<B> {
     ///
     /// # Errors
     /// `error::Unspecified` if inputs not verified.
+    ///
+    /// # FIPS
+    /// * RSA Key Sizes: 1024, 2048, 3072, 4096
+    /// * NIST Elliptic Curves: P256, P384, P521
+    /// * Digest Algorithms: SHA1, SHA256, SHA384, SHA512
```

Comment:
Should we document about FIPS on the individual static values (e.g. `RSA_PKCS1_2048_8192_SHA512`) that are approved?

Reply:
I didn't do it on the top-level for two reasons. It's context specific depending on usage of signing versus verifying, and I wasn't sure how many people would click through since the first sentence of a constant gets embedded into the top-level documentation page.